src/script/standard.h

```diff
@@ -36,11 +36,11 @@ static const unsigned int MAX_OP_RETURN_RELAY = 83;
 
 /**
  * A data carrying output is an unspendable output containing data. The script
- * type is designated as TX_NULL_DATA.
+ * type is designated as NULL_DATA.
  */
 extern bool fAcceptDatacarrier;
 
-/** Maximum size of TX_NULL_DATA scripts that this node considers standard. */
+/** Maximum size of NULL_DATA scripts that this node considers standard. */
```

Comment:
nit:
```suggestion
/** Maximum size of TxoutType::NULL_DATA scripts that this node considers standard. */
```

Reply:
I think in comments we don't need to be overly verbose. Especially not in the file that defines these values. I think the comment is clearer as is.